create_jenkins_job.py

```diff
@@ -168,6 +168,8 @@ def create_job(os_name, job_name, template_file, additional_dict):
 
     # configure os specific jobs
     for os_name in sorted(os_configs.keys()):
+        if os_name != 'linux-armhf':
+            continue
```

Comment:
Doh, this wasn't support to be part of the PR. Removed in a8f4cb5f3cba00d272082a910293c5c60a3cb069.